exe/Plugins.hs

```diff
@@ -183,13 +183,13 @@ idePlugins recorder includeExamples = pluginDescToIdePlugins allPlugins
       Splice.descriptor "splice" :
 #endif
 #if alternateNumberFormat
-      AlternateNumberFormat.descriptor pluginRecorder "alternateNumberFormat" :
+      AlternateNumberFormat.descriptor pluginRecorder :
```

Comment:
@wz1000 for reference this is what I was discussing in IRC. It seems like there are no dupe checks anyway so this should be a safe change?